core/services/document_service.py

```diff
@@ -151,6 +159,7 @@ async def ingest_text(self, request: IngestTextRequest, auth: AuthContext) -> Do
             },
         )
         logger.info(f"Created text document record with ID {doc.external_id}")
+        doc.system_metadata["content"] = request.content
```

Comment:
If this change is necessary, i.e. getting the full content both for text and files, then maybe we should project out this field in the retrieval pipeline because we return the whole doc rn....